src/components/__tests__/setupTests.js

```diff
@@ -2,4 +2,56 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import { RecoilRoot } from 'recoil'
+import userEvent from '@testing-library/user-event'
+
+//components
+import SearchBox from '../SearchBox/SearchBox'
```

Comment:
I actually meant for you to create a unique test file for the search box. Could you move this into its own file? Or really just change the name of this file to searchBox.test.js or something like that. 

Since it is something global I think it can just live in the root of the tests folder. (We can do nesting for more specific component tests, ex. dashboard components)

Reply:
Moved! Submitting another PR now